test/controllers/api/stats_controller_test.rb

```diff
@@ -32,7 +32,7 @@ class Api::StatsControllerTest < ActionDispatch::IntegrationTest
       [3.days.ago.to_date.to_s, 0],
       [2.days.ago.to_date.to_s, 0],
       [1.days.ago.to_date.to_s, 1],
-      [0.days.ago.to_date.to_s, 21]
+      [0.days.ago.to_date.to_s, 22]
```

Comment:
@yachtcaptain23 is there a statement we use to replace the hardcoded value here and above so we don't need to increment this value every time a new publisher fixture is added?